### DISCUSSION TITLE
After updating firmware, M2 solid blue and the other LED remains off.

### DISCUSSION THREAD

**Discussion by oliviaengg** [bitcraze/discussions#1670] [Q&A]
Our crazyfile bolt crashed when testing. Since then, the radio has not able to connect using either the VM crazyfile client or Python command console. During this whole process, the M1 LED has not been on while M2/3/4 were all blinking. 
Then we used the USB cable to power the bolt by connecting the power to a computer, and used the crazyradio to update the firmware via the VM client, which was successfully done. 
After that, however, the M2 stayed solid blue, while M1/3/4 haven't been on since. 
There are no suspicious hot spots on the bolt board.

Thanks in advance if anyone can help.

**Comment by ataffanel** [comment]
This might indicate that the STM32 is not starting. 

The first things to check is that the STM32 can start in bootloader mode by disconnecting the battery, pressing the button and connecting the battery while the button is pressed. When the M2 LED blinks you can release the button.

 - If both blue LEDs blinks, the STM32 is likely healthy and you can flash your bolt using the latest client
 - If only M2 is blinking, there is a problem with the STM32 system and this will need more investigations.
